modeling.py

```diff
@@ -0,0 +1,146 @@
+import nimfa
+import pandas as pd
+from nfldb_tables import NFLFrames
+from scipy.sparse import csr_matrix as csrm
+
+
+class PositionNMF(object):
```

Comment:
@dansbecker I'm guessing that github is not the venue in which to communicate in real time, but I'm currently working on getting a MVP for validation running. If I push up what I have right now, possibly it'll look like the class seems useful. I don't disagree that these things could be split out into functions though.